### ISSUE TITLE
(apprunner-alpha): (Allow specify custom health endpoint)

### ISSUE THREAD

**Issue by BosAtWork** [aws/aws-cdk#26972]
### Describe the feature

By using the new apprunner alpha construct it is currently not possible to change the health configuration ourself.

This is the old apprunner health configuration: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_apprunner.CfnService.HealthCheckConfigurationProperty.html

But in the apprunner alpha package https://docs.aws.amazon.com/cdk/api/v2/docs/aws-apprunner-alpha-readme.html there is now way to change the health endpoint that should be used.

### Use Case

I want to move a Spring Boot application from on-premise to apprunner. This already has an health endpoint in use called /actuator/health. I would like to use this for AppRunner instead of just the root path.

I want to use the alpha apprunner construct as it allows me to upload a docker asset as part of the CDK to make it easier to migrate to cloud for our existing applications.

### Proposed Solution

Allow health configuration to be set in the apprunner alpha construct

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.93.0

### Environment details (OS name and version, etc.)

typescript

**Comment by pahud** [comment]
Yes we'll need to support healthConfiguration in the service L2 construct.

We welcome any pull requsts for that.

**Comment by go-to-k** [comment]
I'll take this issue.
